integration-tests/soak/tests/ocr_test.go

```diff
@@ -51,13 +51,13 @@ var _ = Describe("OCR Soak Test @soak-ocr", func() {
 		})
 
 		By("Setting up Soak Test", func() {
-			chainClient, err := blockchain.NewEthereumMultiNodeClientSetup(soakNetwork)(testEnvironment)
+			chainClient, err := blockchain.NewEVMClient(soakNetwork, testEnvironment)
 			Expect(err).ShouldNot(HaveOccurred(), "Connecting to blockchain nodes shouldn't fail")
 			ocrSoakTest = testsetups.NewOCRSoakTest(&testsetups.OCRSoakTestInputs{
 				BlockchainClient:     chainClient,
 				TestDuration:         time.Minute * 15,
 				NumberOfContracts:    2,
-				ChainlinkNodeFunding: big.NewFloat(1),
+				ChainlinkNodeFunding: big.NewFloat(.1),
```

Comment:
is it intentional?

Reply:
Yes. The new soak test default is just 15 minutes, so only a little funding is needed. The new shorter default is to help catch config errors quicker.